index.html

```diff
@@ -50,18 +37,202 @@ <h1>I'm Salwa <br>Glad to see you!</h1>
+            <div class="tags">
+                <ul>
+                    <li class="tag-pl">html</li>
+                    <li class="tag-pl">css</li>
+                    <li class="tag-pl">javaScript</li>
+                </ul>
+            </div>
+            <div class="btn">
+                 <button type="button">See Project</button>  
+            </div>
+        </div>
+    </div>
+    <div class="card1">
+        <div class="snapshoot">
+            <img src="./style/img/Snapshoot Portfolio3.png" alt="Snapshoot Portfolio.png" class="snapshoot-pic">
+        </div> 
+        <div class="left-block">
+            <div class="tonic-title-frame1">
+                <h2>Tonic</h2>
+                <ul class="tonic-ul">
+                    <li class="client">Canopy</li>
+                    <li class="role">Back End Dev</li>
+                    <li class="year">2015</li>
+                </ul>
+            </div>
+            <div class="text-desc">
+                <p>A daily selection of privately personalized reeds; no accounts or sign-ups required</p>
+            </div>
+            <div class="tags">
+                <ul>
+                    <li class="tag-pl">html</li>
+                    <li class="tag-pl">css</li>
+                    <li class="tag-pl">javaScript</li>
+                </ul>
+            </div>
+            <div class="btn">
+                 <button type="button">See Project</button>  
+            </div>
+        </div>
+    </div>
+    <div class="card1">
+        <div class="snapshoot">
+            <img src="./style/img/Snapshoot Portfolio4.png" alt="Snapshoot Portfolio.png" class="snapshoot-pic">
+        </div> 
+        <div class="left-block">
+            <div class="tonic-title-frame2">
+                <h2>Multi-Post <br>Stories</h2>
+                <ul class="tonic-ul">
+                    <li class="client">Canopy</li>
+                    <li class="role">Back End Dev</li>
+                    <li class="year">2015</li>
+                </ul>
+            </div>
+            <div class="text-desc">
+                <p>A daily selection of privately personalized reeds; no accounts or sign-ups required</p>
+            </div>
+            <div class="tags">
+                <ul>
+                    <li class="tag-pl">html</li>
+                    <li class="tag-pl">css</li>
+                    <li class="tag-pl">javaScript</li>
+                </ul>
+            </div>
+            <div class="btn">
+                 <button type="button">See Project</button>  
+            </div>
         </div>
+    </div>
     </section>
-</body>
-</html>
\ No newline at end of file
+    
+    <!--About myself-->
+    <section class="about">                                                                                                                                                                                      
+        <div class="title">
+            <h1>About<br>Myself</h1>
+        </div>
```

Comment:
- **[OPTIONAL]** I strongly suggest you follow the given design template and apply all the spacing, margins and etc 📝. Also, consider the `border-radius` of the About my self section as you can see below 👇🏽👇🏽
# Your Design
![image](https://user-images.githubusercontent.com/43172164/198839335-02698d62-d5d5-471d-b8de-4e0218cf789c.png)

# Figma Template
![image](https://user-images.githubusercontent.com/43172164/198839084-25c2f12a-ac7b-4629-815a-61b42f224935.png)